pkgs/applications/networking/browsers/misc/widevine-cdm.nix

```diff
@@ -1,34 +1,92 @@
-{ lib
-, stdenv
-, fetchzip
+{
+  lib,
+  stdenv,
+  fetchurl,
+  fetchzip,
+  fetchFromGitHub,
+  squashfsTools,
+  curl,
+  python3,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "widevine-cdm";
   version = "4.10.2830.0";
+  lacrosVersion = "120.0.6098.0";
```

Comment:
For future updates: where are new versions announced?

Reply:
It seems we might just have to look at the widevine installer repo from Asahi Linux, I cannot figure out where officially to find new versions.